rules/android/lint/lint_sources.bzl

```diff
@@ -64,6 +65,11 @@ lint_sources = rule(
             default = [],
             aspects = [collect_aar_aspect],
         ),
+        "lint_checks": attr.label_list(
```

Comment:
It is worth restricting the type of labels that can be supplied here. Leaving this empty might cause wrong jars to be passed unnecessarily. Preferably we can have a custom provider and allow targets only if they provide it.